### DISCUSSION TITLE
Popp 701202 Mold Detector description file limited to firmware v1.0, why ?

### DISCUSSION THREAD

**Discussion by ScratMan** [zwave-js/zwave-js#2517] [Q&A]
Hello,

The Popp 701202 Mold Detector description file specifies the firmware v1.0 only. 
https://github.com/zwave-js/node-zwave-js/blob/fee80cf880c4620f1424fb7c38c7322fcc88ac32/packages/config/config/devices/0x0154/pope701202_1.0-1.0.json

Why is it limited ? I own 4 of them with fw v1.2, hence they aren't recognized by my ZWave-JS, so it's difficult to configure it.

Could we remove the limitation ? should we make some tests first ? I'm running zwavejs2mqtt 3.5.0 and zwave-js 7.2.4 under docker, may I change the file locally to test ?

Thanks for your help

**Comment by AlCalzone** [comment]
It has probably been that way in the source we imported from. I don't see a need to keep the limitation, unless there's a different hardware revision with the same device identifiers.

**Comment by ScratMan** [comment]
I copied the json file from the docker volume using the following command :
`docker cp zwave-js:/usr/src/app/node_modules/@zwave-js/config/config/devices/0x0154/pope701202_1.0-1.0.json .`

Then I modified the firmware version section from 
`	"firmwareVersion": {
		"min": "1.0",
		"max": "1.0"
	},`
to :
`	"firmwareVersion": {
		"min": "0.0",
		"max": "255.255"
	},`

and copied back the file into docker :
`docker cp pope701202_1.0-1.0.json zwave-js:/usr/src/app/node_modules/@zwave-js/config/config/devices/0x0154/`

After restarting the docker container, my 4 Popp Mold detectors were correctly recognized, but lacking the "Configuration" interface.
So I tried to click on "Advanced" and re-interview the corresponding nodes, and the configuration interface is now back 👍 
Even the "[26-112-0-4] Periodic Reports" parameter that showed a negative value in the previous ZWave (deprecated) integration in Home Assistant, now shows the correct default 43200 value 👍👍👍

So I think we can definitely remove the firmware range limitation from the file for future revisions of Zwave-JS.

**Reply by AlCalzone** [reply]
Care to provide a PR? Editing that is possible through github, so it should be quick.

**Reply by ScratMan** [reply]
PR #2521 done.
